### DISCUSSION TITLE
Is the audio that is being converted coming from YouTube videos?

### DISCUSSION THREAD

**Discussion by Bruno5667** [deepjyoti30/ytmdl-web-v2#35] [General]
Is the audio that is being converted coming from YouTube videos?

**Comment by deepjyoti30** [comment]
Yes, the audio is extracted from Youtube videos. You can take a look at the underlying program for the webapp here: https://github.com/deepjyoti30/ytmdl

**Comment by Bruno5667** [comment]
Thank You my friend for the information. I've been looking for a way to
extract mp3 audio from streaming services like Spotify, SoundCloud, or
iTunes that don't come from a video source. I don't have a PC to install
software that will allow me to do that. So I would like to know if there is
something I could use on my phone to extract mp3 audio. I have a Motorola g
stylus using Android 11. If you have any suggestions could you please let
me know. Thanks again.

**Reply by deepjyoti30** [reply]
I don't really have any suggestions on how to do that.

I can suggest going the [awesome-piracy](https://github.com/Igglybuff/awesome-piracy#music) route considering you can use torrents on phone.
